pos_picking_delayed/tests/test_module.py

```diff
@@ -24,21 +25,21 @@ def setUp(self):
                 'base': 'list_price',
             })]
         })
-        # Create a new pos config and open it
-        self.pos_config = self.env.ref('point_of_sale.pos_config_main').copy()
+        self.pos_config = self.env.ref('point_of_sale.pos_config_main')
```

Comment:
Why did you removed .copy() ?

Reply:
I had some issues with some defaults not beign applied or provoking problems, so I found that in the end having the default pos_config was more stable after all.

Reply:
The copy allow to use DB, and run test, during development. (Otherwise, if the session is opened, the tests fail), so I'd like to keep it, _if possible_.
I dont understand how copy config could break default value, could you be more precise ?
Thanks.

Reply:
Sylvain you were too fast! :sweat_smile: :   I just wanted to retest my CIs and in this manner they still fail. I'd say I that in order to keep the copy() when the instance currency is different from the demo data one I have to assign the pricelist to both of them... I'll test in another PR :smile: